pkgs/development/python-modules/python-engineio/3.nix

```diff
@@ -0,0 +1,62 @@
+{ lib, stdenv
+, buildPythonPackage
+, fetchFromGitHub
+, aiohttp
+, eventlet
+, iana-etc
+, libredirect
+, mock
+, requests
+, six
+, tornado
+, websocket_client
+, websockets
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "python-engineio";
+  version = "3.14.2";
+
+  src = fetchFromGitHub {
+    owner = "miguelgrinberg";
+    repo = "python-engineio";
+    rev = "v${version}";
+    sha256 = "1r3gvizrknbv036pvxid1l726wkb0l43bdaz5y879s7j3ipyb464";
+  };
+
+  checkInputs = [
+    aiohttp
+    eventlet
+    mock
+    requests
+    tornado
+    websocket_client
+    websockets
+    pytestCheckHook
+  ];
+
+  doCheck = !stdenv.isDarwin;
```

Comment:
Why do we not want to run this on darwin?

Reply:
This was copied from pkgs/development/python-modules/python-engineio/default.nix, I don't know if it works or not but it seems like checks are enabled for engineio 3.x in 20.09, so I'll remove this.